storage/src/test/java/org/apache/kafka/storage/internals/log/LogValidatorTest.java

```diff
@@ -2087,7 +2076,6 @@ public void checkLogAppendTimeNonCompressed(byte magic) {
                 expectedMaxTimestampOffset = 2;
                 break;
         }
-        assertEquals(expectedMaxTimestampOffset, validatedResults.shallowOffsetOfMaxTimestamp);
```

Comment:
`expectedMaxTimestampOffset` is no longer used and should be removed.

Reply:
Removed it. Thanks.